### DISCUSSION TITLE
g0v tw hackath62n — 提案 — #design ／ Tofus — 帶我去月球黑客松

### DISCUSSION THREAD

**Discussion by leechiuhui** [3dw/freemath#75] [General]
https://youtu.be/Fdzx-_8_3w4?si=ICEtJEWD5_1perbs
Freemath 也在坑上被當範例了

**Comment by bestian** [comment]
切豆腐的概念很生動, 太大塊的就吃不動了. 小塊小塊就可以一人做一點.
